UPGRADE.md

```diff
@@ -52,7 +52,7 @@ There is a list of all the repositories maintained by monorepo, changes in log b
 ## [From v7.0.0-beta4 to Unreleased]
 ### [shopsys/project-base]
 - [#633 Google Cloud deploy using Terraform, Kustomize and Kubernetes](https://github.com/shopsys/shopsys/pull/633)
-    - update your `.dockerignore` to ignore infrastructure files follow [changes](https://github.com/shopsys/shopsys/pull/633/commits/5e507aa0aff44cb689b8d65fba58da53a8fafd1f)
+    - update your `.dockerignore` to ignore infrastructure files, follow [changes](https://github.com/shopsys/shopsys/pull/633/commits/5e507aa0aff44cb689b8d65fba58da53a8fafd1f)
```

Comment:
commit id will not be the same after squash

Reply:
@MattCzerner this is a valid point, will you take care of it during final rebase?